cgroups/fs/cpuset.go

```diff
@@ -17,12 +17,9 @@ type CpusetGroup struct {
 func (s *CpusetGroup) Apply(d *data) error {
 	dir, err := d.path("cpuset")
 	if err != nil {
-		if cgroups.IsNotFound(err) {
-			return nil
-		} else {
-			return err
-		}
+		return err
```

Comment:
The rest of the cgroup setups use data.path() and cpuset does something different. This change makes sure it stays in sync.